rules/linux/discovery_suid_sguid_enumeration.toml

```diff
@@ -0,0 +1,73 @@
+[metadata]
+creation_date = "2023/07/24"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/07/24"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule monitors for the usage of the "find" command in conjunction with SUID and SGUID permission arguments. SUID 
+(Set User ID) and SGID (Set Group ID) are special permissions in Linux that allow a program to execute with the 
+privileges of the file owner or group, respectively, rather than the privileges of the user running the program. 
+Attackers can leverage these permissions to escalate privileges by exploiting vulnerabilities or built-in features 
+in the privileged program.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*", "endgame-*"]
+language = "eql"
+license = "Elastic License v2"
+name = "SUID/SGUID Enumeration Detected"
+risk_score = 21
+rule_id = "5b06a27f-ad72-4499-91db-0c69667bffa5"
+severity = "low"
+tags = ["Domain: Endpoint", "OS: Linux", "Use Case: Threat Detection", "Tactic: Discovery", "Tactic: Privilege Escalation", "Data Source: Elastic Endgame"]
+timestamp_override = "event.ingested"
+type = "eql"
+query = '''
+process where host.os.type == "linux" and event.type == "start" and event.action in ("exec", "exec_event") and 
+process.name == "find" and process.args : "-perm" and process.args : (
+"/6000", "-6000", "/4000", "-4000", "/2000", "-2000", "/u=s", "-u=s", "/g=s", "-g=s", "/u=s,g=s", "/g=s,u=s"
+) and not user.Ext.real.id : "0" and not group.Ext.real.id : "0"
```

Comment:
Does Endgame populate these fields?

Reply:
You're right, I was a bit too quick on that one.